Source/EasyNetQ/EasyNetQ.csproj

```diff
@@ -28,5 +28,6 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
```

Comment:
Does System.Diagnostics.DiagnosticSource act as cross-platform replacement of System.Diagnostics.Tracing.EventSource ?